packages/image_picker/image_picker_windows/lib/image_picker_windows.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:file_selector_windows/file_selector_windows.dart';
+import 'package:flutter/foundation.dart';
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+
+/// The Windows implementation of [ImagePickerPlatform].
+///
+/// This class implements the `package:image_picker` functionality for
+/// Windows.
+class ImagePickerWindows extends ImagePickerPlatform {
+  /// Constructs a ImagePickerWindows.
+  ImagePickerWindows();
+
+  /// List of image extensions used when picking images
+  @visibleForTesting
+  static const List<String> imageFormats = <String>[
+    'jpg',
+    'jpeg',
+    'png',
+    'bmp',
+    'webp',
+    'gif',
+    'tif',
+    'tiff',
+    'apng'
+  ];
+
+  /// List of video extensions used when picking videos
+  @visibleForTesting
+  static const List<String> videoFormats = <String>[
+    'mov',
+    'wmv',
+    'mkv',
+    'mp4',
+    'webm',
+    'avi',
+    'mpeg',
+    'mpg'
+  ];
+
+  /// The FileSelectorPlatform used by the ImagePickerWindows.
+  @visibleForTesting
+  static late FileSelectorPlatform fileSelectorInstance = FileSelectorWindows();
```

Comment:
Consider dropping `Instance` from the name; it doesn't add any information, since all variables are instances of some type. Plus it might save a line-wrap some day :)